source/position3dlabel.ts

```diff
@@ -0,0 +1,124 @@
+
+import { mat4, vec3 } from 'gl-matrix';
+
+import { FontFace } from './fontface';
+import { GlyphVertices } from './glyphvertices';
+import { Label } from './label';
+import { Text } from './text';
+import { Typesetter } from './typesetter';
+
+
+/**
+ * A Label that can be positioned in 3D space.
+ * The unit for positions, size and transformations, is the abstract World Unit.
+ */
+export class Position3DLabel extends Label {
+
+    protected _position: vec3;
+    protected _direction: vec3;
+    protected _up: vec3;
+
+    /**
+     * Constructs a pre-configured 3D-label with given text.
+     * @param text - Valid context to create the object for.
+     * @param identifier - Meaningful name for identification of this instances VAO and VBOs.
+     */
+    constructor(text: Text, fontFace: FontFace) {
+        super(text, fontFace);
+        this._position = vec3.fromValues(0, 0, 0);
```

Comment:
I prefer floating point notation for non integer semantic, `vec3.fromValues(0.0, 0.0, 0.0);` same applies below...